docs/operations.rst

```diff
@@ -51,3 +51,39 @@ The important bits here are:
 Backups
 *******
 
+Performing regular backups of the state store is an important operational component for any production deployment of ``alchemiscale``.
+To do this, **first shut down the ``neo4j`` service so that no database processes are currently running**.
+
+The instructions below assume a Docker-based deployment, perhaps via ``docker-compose`` as in :ref:`deploy-docker-compose`.
+The same general principles apply to any deployment type, however.
+
+Creating a database dump
+========================
+
+**With the neo4j service shut down**, navigate to the directory containing your database data, set ``$BACKUPS_DIR`` to the absolute path of your choice, then run::
+
+    docker run --rm \
+               -v $(pwd):/var/lib/neo4j/data \
+               -v ${BACKUPS_DIR}:/tmp \
+               --entrypoint /bin/bash \
+               neo4j:4.4 \
```

Comment:
We should add a note this version should match the tag of the container used to run the database

Reply:
Ah yes, good point! Adding this.

Reply:
Done!